src/app/static/src/app/components/plots/table/Table.vue

```diff
@@ -45,7 +45,7 @@ export default defineComponent({
     },
     setup(props) {
         const store = useStore<RootState>();
-        const plotData = computed<PlotData>(() => store.state.plotData[props.plotName]);
+        const plotData = computed<SquarePlotData>(() => store.state.plotData[props.plotName] as SquarePlotData);
```

Comment:
I think we may need to change the type for the data prop in TableReshapeData.vue to match with the new SquarePlotData type. Getting a TS compiler error in this file on line 3.

Reply:
Looks good, no compiler errors now. Agreed on the name change as well